### PR TITLE
UBI image update

### PR DESCRIPTION
**Pull request by gayathrimenath-ibm**

### CODE REVIEW

**Review by akgunjal — APPROVED** [review]
LGTM

File: config-templates/redhat/odf-local/changelog.json [hunk 1]
```diff
@@ -3,19 +3,19 @@
         "4.10": {
             "status": "supported",
             "latest-revision": {
-                "revision": "11",
-                "release-date": "09 January 2023",
-                "cves": "CVE-2022-41717,CVE-2022-41720",
+                "revision": "12",
+                "release-date": "19 January 2023",
```
Comment (mssachan) [inline config-templates/redhat/odf-local/changelog.json]: Change date to `23 Jan`

File: config-templates/redhat/odf-local/changelog.json [hunk 2]
```diff
@@ -3,19 +3,19 @@
...
+                "cves": "",
                 "golang": "1.18.9",
-                "ubi": "8.7-923.1669829893",
+                "ubi": "8.7-1049",
                 "new-features": [],
-                "fixes": []
+                "fixes": ["UBI image update"]
```
Comment (mssachan) [inline config-templates/redhat/odf-local/changelog.json]: No need to write `UBI image update`. Keep it empty

File: config-templates/redhat/odf-local/changelog.json [hunk 3]
```diff
@@ -3,19 +3,19 @@
         "4.10": {
             "status": "supported",
             "latest-revision": {
-                "revision": "11",
-                "release-date": "09 January 2023",
-                "cves": "CVE-2022-41717,CVE-2022-41720",
+                "revision": "12",
+                "release-date": "19 January 2023",
+                "cves": "",
```
Comment (mssachan) [inline config-templates/redhat/odf-local/changelog.json]: Add all CVEs which are being fixed with the new revision

File: config-templates/redhat/odf-remote/changelog.json [hunk 4]
```diff
@@ -3,19 +3,19 @@
         "4.10": {
             "status": "supported",
             "latest-revision": {
-                "revision": "11",
-                "release-date": "09 January 2023",
-                "cves": "CVE-2022-41717,CVE-2022-41720",
+                "revision": "12",
+                "release-date": "19 January 2023",
```
Comment (mssachan) [inline config-templates/redhat/odf-remote/changelog.json]: Same here for all versions

**Review by mssachan — CHANGES_REQUESTED** [review]

**Review by mssachan — APPROVED** [review]
LGTM

**Pull request merged**